documentation/migration-guide-0.1.1-to-0.2.0.md

```diff
@@ -0,0 +1,30 @@
+# Migration guide from v0.1.1 to v0.2.0
+
+### Added seed job agent
+Now seed jobs are not built by master executors, but by dedicated agents deployed to Kubernetes. We disabled master executors for security reasons.
+
+### Apply Jenkins configuration via Groovy scripts instead of Jenkins jobs
+We have removed hardcoded configuration by Jenkins jobs. 
+In v0.1.1 jenkins-operator configuration was stored in `jenkins-operator-user-configuration-<cr_name>`
+If you want to use v0.2.0 or newer you must simply write refer to old ConfigMap by modifying CRD, for example:
+
+```yaml
+apiVersion: jenkins.io/v1alpha2
+kind: Jenkins
+metadata:
+  name: example
+spec:
+  configurationAsCode:
+    configurations: 
+    - name: jenkins-operator-user-configuration
+  groovyScripts:
+    configurations:
+    - name: jenkins-operator-user-configuration
+```
+
+Jenkins configuration jobs are removed from core and UI. 
+In v0.1.1 you could see if configuration failed or successfully updated in UI in job logs. 
+Now, when Jenkins configuration jobs are removed, you must use this command to see that configuration fail or success.
```

Comment:
Now, when Jenkins configuration jobs are removed, you must use this command to see if configuration was failed.